Subject: [PATCH] Append pwd in "INCLUDE" env var relative paths
---
Index: cargo/private/cargo_build_script.bzl
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/cargo/private/cargo_build_script.bzl b/cargo/private/cargo_build_script.bzl
--- a/cargo/private/cargo_build_script.bzl	(revision cdaf15f5796e3e934b074526272823284bbaed01)
+++ b/cargo/private/cargo_build_script.bzl	(revision 87bceac16dea3af67c7643abd8a72d2e0a8dd0e2)
@@ -232,6 +232,23 @@
             res.append(arg)
     return res

+def _pwd_paths(args):
+    """Prefix execroot-relative paths of known arguments with ${pwd}.
+
+    Args:
+        args (list): List of tool arguments.
+
+    Returns:
+        list: The modified argument list.
+    """
+    res = []
+    for path in args:
+        if not paths.is_absolute(path):
+            res.append("${{pwd}}/{}".format(path))
+        else:
+            res.append(path)
+    return res
+
 def _pwd_flags(args):
     return _pwd_flags_fsanitize_ignorelist(_pwd_flags_isystem(_pwd_flags_bindir(_pwd_flags_sysroot(args))))

@@ -432,7 +449,7 @@
         cc_c_args, cc_cxx_args, cc_env = get_cc_compile_args_and_env(cc_toolchain, feature_configuration)
         include = cc_env.get("INCLUDE")
         if include:
-            env["INCLUDE"] = include
+            env["INCLUDE"] = ";".join(_pwd_paths(include.split(";")))

         toolchain_tools.append(cc_toolchain.all_files)

