Subject: [PATCH] Force musl
---
Index: rust/platform/triple_mappings.bzl
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/rust/platform/triple_mappings.bzl b/rust/platform/triple_mappings.bzl
--- a/rust/platform/triple_mappings.bzl	(revision 2678bf75a0bfaa9088a98acda8ffb2d4399cbd3d)
+++ b/rust/platform/triple_mappings.bzl	(revision a8860c8f62ed24b612852847c4dc3988138821a4)
@@ -47,6 +47,7 @@
     "aarch64-linux-android": _support(std = True, host_tools = False),
     "aarch64-pc-windows-msvc": _support(std = True, host_tools = True),
     "aarch64-unknown-fuchsia": _support(std = True, host_tools = False),
+    "aarch64-unknown-linux-musl": _support(std = True, host_tools = False),
     "aarch64-unknown-uefi": _support(std = True, host_tools = False),
     "arm-unknown-linux-gnueabi": _support(std = True, host_tools = True),
     "armv7-linux-androideabi": _support(std = True, host_tools = False),
@@ -69,6 +70,7 @@
     "x86_64-linux-android": _support(std = True, host_tools = False),
     "x86_64-unknown-freebsd": _support(std = True, host_tools = True),
     "x86_64-unknown-fuchsia": _support(std = True, host_tools = False),
+    "x86_64-unknown-linux-musl": _support(std = True, host_tools = False),
     "x86_64-unknown-none": _support(std = True, host_tools = False),
     "x86_64-unknown-uefi": _support(std = True, host_tools = False),
 }
