Subject: [PATCH] Verbose Rustc flag
---
Index: rust/private/rustc.bzl
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/rust/private/rustc.bzl b/rust/private/rustc.bzl
--- a/rust/private/rustc.bzl	(revision 3a83745b342855896081296bb43359018435433a)
+++ b/rust/private/rustc.bzl	(revision 141c29af88ec18361259cedab54eb8c1ae1a0d0e)
@@ -1015,10 +1015,11 @@
     # Rustc arguments
     rustc_flags = ctx.actions.args()
     rustc_flags.set_param_file_format("multiline")
-    rustc_flags.use_param_file("@%s", use_always = False)
+    rustc_flags.use_param_file("@%s", use_always = True)
     rustc_flags.add(crate_info.root)
     rustc_flags.add(crate_info.name, format = "--crate-name=%s")
     rustc_flags.add(crate_info.type, format = "--crate-type=%s")
+    rustc_flags.add("--verbose")

     if error_format == None:
         error_format = get_error_format(attr, "_error_format")
